src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -112,7 +112,7 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
-            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (self::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
```

Comment:
Hmm… that won't be so easy since we're inside a closure, which is itself inside another closure. But the condition to run this piece of code is just to change the error handler. Let's try that.